openfisca_france/model/prestations/aide_permis_probtp.py

```diff
@@ -0,0 +1,35 @@
+from openfisca_france.model.base import Variable, Individu, MONTH
+from numpy import select
+
+
+class aide_permis_probtp(Variable):
+    value_type = float
+    entity = Individu
+    label = "Montant de l'aide au permis B PROBTP"
+    reference = ["https://www.probtp.com/part/apprenti/aide-permis.html"]
+    definition_period = MONTH
+
+    def formula(individu, period, parameters):
+        params = parameters(period).prestations.aide_permis_probtp
+
+        rfr = individu.foyer_fiscal('rfr', period.this_year)
+        montant = select(
+            [
+                rfr <= params.ressources_annuelles_maximal.premier_echelon,
+                rfr <= params.ressources_annuelles_maximal.deuxieme_echelon,
+                rfr <= params.ressources_annuelles_maximal.troisieme_echelon,
+                ],
+            [
+                params.montant.premier_echelon,
+                params.montant.deuxieme_echelon,
+                params.montant.troisieme_echelon,
+                ],
+            default=0
+            )
+
+        age = individu('age', period)
+        eligibilite_age = (params.age.minimum <= age) * (age <= params.age.maximum)
+
+        alternant = individu("alternant", period)
+
+        return montant * eligibilite_age * alternant
```

Comment:
Privilégier les conditions d'éligibilité sur le montant ?
```suggestion
        return eligibilite_age * alternant * montant
```

Reply:
Je ne savais pas qu'il y avait un ordre au niveau des variables dans notre cas présent les variables d'éligibilités doivent se placer avant celle de montant. Je change ça.